### PR TITLE
rubocop update

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: src/clients/nfs-client4part.rb [hunk 1]
```diff
@@ -62,8 +62,6 @@ def main
 
         if Builtins.contains(@w_ids, Convert.to_symbol(@widget_id))
           return ToStorage(@modify_line)
-        else
-          return nil
```
Comment (lslezak) [inline src/clients/nfs-client4part.rb]: The question is if we should explicitly return `nil` in this case. That `nil` default might be confusing for someone...

What about:
```
Builtins.contains(@w_ids, Convert.to_symbol(@widget_id)) ? ToStorage(@modify_line) : nil
```
or if the line is too long:
```
is_included = Builtins.contains(@w_ids, Convert.to_symbol(@widget_id))
is_included ? ToStorage(@modify_line) : nil
```

Comment (jreidinger) [inline src/clients/nfs-client4part.rb]: see https://github.com/yast/yast-nfs-client/pull/51/files/3dccf898d3c2490de1cbae4d9588db32d2dee760#diff-9c47915e3147e857bb6467c6946cb7a6L70

it already returning nil explicitelly

Comment (lslezak) [inline src/clients/nfs-client4part.rb]: OK, overlooked that the method continues.. :wink:

**Comment by lslezak** [comment]
LGTM

**Comment by lslezak** [comment]
Thanks, still LGTM :wink:

**Pull request merged**